packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/hooks/useFormatTimeStamp.js

```diff
@@ -0,0 +1,15 @@
+import { useIntl } from 'react-intl';
+
+const useFormatTimeStamp = (value) => {
+  const { formatDate } = useIntl();
+
+  const formattedDateTime = formatDate(new Date(value), {
+    dateStyle: 'long',
+    timeStyle: 'medium',
+    hourCycle: 'h24',
+  });
+
+  return formattedDateTime.replace(/\s+at/, ',');
```

Comment:
This may cause a problems in languages other than english. Would there be a way to get the format we want directly from `formatDate`?

Reply:
I couldn't find one from formatDate or js intl, but for now retrieving separately and concatenating to avoid language issue here. If you have any other suggest, please suggest.